sunnypilot/models/fetcher.py

```diff
@@ -118,7 +118,7 @@ def set(self, data: dict) -> None:
 
 class ModelFetcher:
   """Handles fetching and caching of model data from remote source"""
-  MODEL_URL = "https://docs.sunnypilot.ai/driving_models.json"
+  MODEL_URL = "https://gitlab.com/sunnypilot/public/docs.sunnypilot.ai/-/raw/update-model-json-mix-tinygrad-and-snpe-models/driving_models_new.json"
```

Comment:
Need to change this before merging, this is the PR https://gitlab.com/sunnypilot/public/docs.sunnypilot.ai/-/merge_requests/22